pkg/operator/k8sutil/pvc.go

```diff
@@ -31,10 +33,24 @@ func ExpandPVCIfRequired(client client.Client, desiredPVC *v1.PersistentVolumeCl
 		logger.Debugf("desired or current size are not specified for PVC %q", currentPVC.Name)
 		return
 	}
+
 	if desiredSize.Value() > currentSize.Value() {
+		// get StorageClass
+		storageClass := &storagev1.StorageClass{}
+		err := client.Get(context.TODO(), types.NamespacedName{Name: *(currentPVC.Spec.StorageClassName)}, storageClass)
```

Comment:
Is the StorageClassName always set on the PVC? Since it can be nil, probably not, so we would need to check for nil first.

Reply:
It should be always set as far as I know. If someone does not set it should pick up the default one in the PVC defaulting

Reply:
Maybe it will never happen, but to be conservative I'd still suggest to check for nil. Even if something shouldn't be nil, if the type is nullable it will probably be nil when we least expect it. :)

Reply:
Done